app/src/command.c

```diff
@@ -32,7 +54,12 @@ process_t adb_execute(const char *serial, const char *const adb_cmd[], int len)
 
     memcpy(&cmd[i], adb_cmd, len * sizeof(const char *));
     cmd[len + i] = NULL;
-    return cmd_execute(cmd[0], cmd);
+    r = cmd_execute(cmd[0], cmd, &process);
+    if (r != 0) {
+        show_err_msg(r);
```

Comment:
If the error is not reported to the caller of `adb_execute()` (I'm ok with this), the relevant error message can be printed directly in the specific implementation.

Reply:
In my opinion, the `cmd_exectue` is a general function. That is, it can execute any cmd. When it errors, problem should be handled by caller (print error msg in this case).

Reply:
Oh, yes, of course. I confused `adb_execute()` and `cmd_execute()` (initially, I thought that even `adb_execute()` would report error to the caller). Whatever. It's better like that.